Detectors/ITSMFT/ITS/macros/test/run_test.sh

```diff
@@ -10,7 +10,7 @@ mcEngine=TGeant3
 #MC event generator
 mcGener=boxgen
 
-o2sim -n $nEvents -e $mcEngine -g $mcGener -m PIPE ITS >& sim_its.log
+o2sim_serial -n $nEvents -e $mcEngine -g $mcGener -m PIPE ITS >& sim_its.log
 
 root -b -q $O2_ROOT/share/macro/run_digi_its.C+\($rate\) >& digi_its.log
```

Comment:
@shahor02 Actually, this change has very little to do with this PR.  It is there just to make the old (FairRunaAna-based) test suite work again.

The current o2sim (the parallel one) creates ROOT files having some number (process ID ?)
used in their names:  for example o2sim_31358_par.root instead of just o2sim_par.root.
This makes the subsequent (old, FairRunaAna-based) macros confused.
So I opted for the easiest:  to just fall back to the o2sim_serial.

Once this PR request is merged and the new (DPL-based) test is in place, 
the old test suite (including this run_test.sh script) can be happily dropped.